app/controllers/posts_controller.rb

```diff
@@ -544,7 +544,8 @@ def create_params
       :reply_to_post_number,
       :auto_track,
       :typing_duration_msecs,
-      :composer_open_duration_msecs
+      :composer_open_duration_msecs,
+      :created_at
```

Comment:
thanks for making this commit! I think we want to make sure this is only permitted if the api is being used. Perhaps we can leave the permitted array as is, but afterwards check if we are using the api (`is_api?`) and then append created_at to the array: `permitted << :created_at`